src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1214,6 +1214,206 @@ namespace Microsoft.Management.Deployment
         String InstallationNotes { get; };
     }
 
+    /// IMPLEMENTATION NOTE: AddPackageCatalogOptions
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 11)]
+    runtimeclass AddPackageCatalogOptions
+    {
+        AddPackageCatalogOptions();
+
+        /// The name of the catalog to add.
+        String CatalogName;

```

Comment:
```suggestion
        String Name;
```
Unless there is to be another name, it seems redundant to call out catalog again.